packages/shared/ReactFeatureFlags.js

```diff
@@ -25,7 +25,7 @@ export const debugRenderPhaseSideEffectsForStrictMode = __DEV__;
 
 // Helps identify code that is not safe for planned Offscreen API and Suspense semantics;
 // this feature flag only impacts StrictEffectsMode.
-export const enableStrictEffects = false;
+export const enableStrictEffects = true;
```

Comment:
```suggestion
export const enableStrictEffects = __DEV__;
```

Reply:
I could do this but there's a small inconsistency in that the WWW version re-exports a GK which does _not_ have a DEV check. I'd rather have them all consistent. Currently I think we always check for `__DEV__` in the source anyway before any behavioral changes. So I'd propose either:

1. Keep it `true`
2. Make it `__DEV__` and change WWW feature flag to be `__DEV__ && FeatureFlagsWWW.enableStrictEffects`

Reply:
I think it makes more sense to compare to other flags _within this feature flags file_ rather than www. So this would be more like `debugRenderPhaseSideEffectsForStrictMode` right above it.

Reply:
> Currently I think we always check for `__DEV__` in the source anyway before any behavioral changes.

Yeah, but this is to enable static code elimination in production builds even if there's a dynamic flag (like in www).

Reply:
>I think it makes more sense to compare to other flags within this feature flags file rather than www.

I'm thinking of a situation where we add some behavior difference in the source without checking `__DEV__` before it. Then, if WWW and OSS flags diverge in this way, you'd see the different behavior in production on WWW than in production in OSS. That seems like a risky divergence.

Reply:
Then let's change the OSS export too.

There's good reason to keep the extra static `__DEV__` wrapper (mentioned [above](https://github.com/facebook/react/pull/21590#discussion_r642549750)) and I think there's also value in keeping the feature flags more scoped/explicit about DEV-only features.